modules/installation/pages/client-requirements.adoc

```diff
@@ -21,15 +21,33 @@ For details on supported product versions, see https://www.suse.com/lifecycle.
 | {sle} 15 | {x86}_64, {ppc}, {ibmz}, ARM               | icon:check[role="green"]      | icon:check[role="green"]
 | {sle} 12 | {x86}_64, {ppc}, {ibmz}, ARM               | icon:check[role="green"]      | icon:check[role="green"]
 | {sle} 11 | {x86}, {x86}_64, {ipf}, IBM {ppc}, {ibmz}  | icon:check[role="green"]      | icon:check[role="green"]
+| {sles}-ES 8 | {x86}_64                                 | icon:check[role="green"]      | icon:check[role="green"]
 | {sles}-ES 7 | {x86}_64                                 | icon:check[role="green"]      | icon:check[role="green"]
 | {sles}-ES 6 | {x86}_64                                 | icon:check[role="green"]      | icon:check[role="green"]
 | {sles} for SAP | {x86}_64, {ppc}                       | icon:check[role="green"]      | icon:check[role="green"]
 | {rhel} 8 | {x86}_64                                   | icon:question[role="gray"]    | icon:question[role="gray"]
 | {rhel} 7 | {x86}_64                                   | icon:check[role="green"]      | icon:check[role="green"]
 | {rhel} 6 | {x86}, {x86}_64                            | icon:check[role="green"]      | icon:check[role="green"]
+| CentOS 8 | {x86}, {x86}_64                            | icon:question[role="gray"]    | icon:question[role="gray"]
 | CentOS 7 | {x86}, {x86}_64                            | icon:question[role="gray"]    | icon:question[role="gray"]
 | CentOS 6 | {x86}, {x86}_64                            | icon:question[role="gray"]    | icon:question[role="gray"]
 | {opensuse} Leap 15.1 | {x86}_64                       | icon:times[role="danger"]      | icon:check[role="green"]
-| {ubuntu} 16.04 | {x86}_64                             | icon:times[role="danger"]      | icon:check[role="green"]
+| {ubuntu} 20.04 | {x86}_64                             | icon:times[role="danger"]      | icon:check[role="green"]
 | {ubuntu} 18.04 | {x86}_64                             | icon:times[role="danger"]      | icon:check[role="green"]
+| {ubuntu} 16.04 | {x86}_64                             | icon:times[role="danger"]      | icon:check[role="green"]
+|===
+
+
+
+For most clients, minimum hardware is required.
```

Comment:
I would suggest to explain more what the management stack need on top of the normal OS and workload.

CPU: the OS needs at least 1. And I think we do not need more just because we want to manage it with salt.
RAM: the OS has minimal and recommended values. Maybe you can check the recommended values of the OSes but I think we do not need to increase it. Be careful with "minimal" values. AFAIK the SLES minimal does not really allow to "update" the client as the metadata of the update channels cause OOM exceptions.
SWAP: just standard recommendation. It is a safety value in case the RAM is not big enough. You can also put RAM+SWAP together as virtual memory. This would also cover s390x
Disk: this is difficult as we do not know what the customer want to run on this. So here I would suggest to say something about the size of the management stack. But even this is not easy. So maybe we can say 200MB on top of a minimal OS installation or something like this.

Reply:
> I would suggest to explain more what the management stack need on top of the normal OS and workload.

In fact, I would *only* say that about client requirements. Something in the lines of "whatever your workload needs, plus 200 MB RAM for Salt or X MB RAM for the traditional stack. No additional CPU core required unless your client already struggles with the workload".